app/middleware/routeHelper.js

```diff
@@ -0,0 +1,10 @@
+const renderer = require('./renderer');
+
+function renderSymptomsPage(req, res, errorMessage) {
```

Comment:
Rather than having the message in the parameter list this function could be renamed to something like `renderSymptomsWithError`. Include the message module in this module and set it.